addons/l10n_mx/data/account_configuration.yml

```diff
@@ -0,0 +1,10 @@
+-
+  Account settings set for Use Anglo-Saxon Accounting
+-
+  !record {model: account.config.settings, id: mx_default_account_properties}:
+    {
+      'use_anglo_saxon': 1,
```

Comment:
Because is illegal not process your costs as anglo_saxon does.

AFAICR @qdp-odoo mentioned that anglo saxon will be migrated inside account and not optional at all (because it is the standard in United sate also).

The point here is that you can always make an external module (a multilocalization environment is far from being managed in saas-x and at least until we finish 10 or 20 localization we will have several of this pointless discussions.

If you are installing l10n_mx it is suposed you want to comply (without human intervention extra) with the legality of México, think otherwise is pointless on this stage of the solution and bring more early problems in the incorrect time.

In the near future this will absolutely change I am sure.